.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
```

Comment:
I think we also need to ask for logs (server, compiler, agent) if they are relevant. Not sure I which section this belongs.